### ISSUE TITLE
Are the parameters given in config_X_mira.yaml same as the training parameters?

### ISSUE THREAD

**Issue by alpercanberk** [mira-space/Mira#3]
If not, could I learn about the training parameters (e.g. effective batch size, learning rate, clipping, etc.)

**Comment by zzyfd** [comment]
Hello, the settings specified in the config_X_mira.yaml file are ready to be used for training Mira on A100 40G GPU.

**Comment by alpercanberk** [comment]
Wait, so the model was trained with 1 A100 and no gradient accumulation?

**Comment by mira-space** [comment]
No, the Mira-v0 model was trained on 32 A100 GPUs for approximately two days.
